---
layout: page
title:
permalink: /research/
---
# Team: Avoidance and Repair of Biological Damage #

This team studies some of the key conceptual questions of biology and evolution, like evolution of mutation rates, and origin and consequences of phenotypic variability. We are also aiming to facilitate the development of new therapeutic strategies, e.g., antibiotic treatments, which will take into account not only mechanistic aspects as is the case today, but also evolutionary and ecological processes responsible for their emergence.

### Ongoing research projects:

1. *__Origin of spontaneous mutations__*. All living organisms must maintain the correct balance between the avoidance of mutations in order to preserve genetic information, and generation of mutations without which there will be no evolution. Such a balance is assured by the activity of a variety of molecular mechanisms. Despite extensive knowledge of these molecular mechanisms, it is very difficult to know which molecular mechanism is responsible for the generation of a spontaneous mutation when it arises in a population of genetically wild-type bacteria. Our goal is to identify the molecular determinants of variations in mutation rates at the single cell level driven by phenotypic variability within populations of genetically identical cells. 

2. *__Origin and evolutionary consequences of erroneous protein synthesis__*. Proteins are essential for all cellular functions. However, in spite of the fact that multiple mechanisms ensuring protein synthesis fidelity exist, translational error rates are orders of magnitude higher than transcription and DNA replication error rates in both prokaryotes and eukaryotes. Translation errors are generally thought to be deleterious to the cell. However, some errors may produce protein variants, which are beneficial for the microbial population. Therefore, translation errors might transiently increase phenotypic variability in isogenic microbial populations without changing their genotype, and thus increase adaptability of microbial populations in fluctuating natural environments but also avoid deleterious effects to be hereditarily transmitted. Our goal is to unravel the molecular and ecological determinants of translation fidelity as well as its evolutionary consequences.
 
3. *__Biological effects of subinhibitory concentrations of antibiotics__*. It is generally believed that clinically relevant antibiotic resistance appears as a consequence of bacterial exposure to concentrations above the minimum inhibitory concentration (MIC) of antibiotics. For this reason, the impact of sub-MICs of antibiotics on bacterial populations has been largely neglected. However, there is growing evidence indicating that sub-MICs of antibiotics may play an important role in the emergence and spread of antibiotic resistance. Furthermore, an emphasis on the therapeutic potential of antibiotics has resulted in a paucity of studies related to the impact that they may have on bacteria other than the evolution of antibiotic resistance. Cells exposed to sub-MICs of antibiotics do not stop growing and are not killed. This suggests that they are able to adjust their gene expression patterns and metabolism to simultaneously respond to the antibiotic-induced stresses and to maintain rapid growth. Our goal is to identify molecular mechanisms underlying these bacterial responses, as well as their impact on evolution of antibiotic resistance.

 
# SEED Team : Systems Engineering and Evolution Dynamics #


The team’s main efforts rely on years of investment in building an intellectual and experimental framework based on interdisciplinary approaches, harnessing physics and computer science and welcoming young researchers from these domains to address key questions in Life Sciences in a quantitative manner using Escherichia coli as the simplest (yet still not fully understood) model organism. 
Recently, our focus was on three main axes:

1. *__Evolution and robustness of cooperative traits__*. We show that horizontal gene transfer evolved to spread public goods-encoding genes and helps maintain cooperation.  The effect is strongest when public goods genes are implicated in epidemic dynamics, making horizontal transfer especially relevant for pathogenic bacteria that repeatedly infect new hosts and base their virulence on costly public goods. This research theme will be discontinued in the coming year apart from our focus on ageing-related cooperation traits.

2. *__Cellular degeneration and ageing__*.  We established E. coli as a relevant model system to study biodemographics and mechanisms of senescence. We introduced novel state-of-the art ‘lab on chip’ microfluidics devices that allow us to follow both chronological and replicative ageing of large populations of single cells. We show that E. coli exhibits a common pattern of exponential increase in mortality rate also observed in higher organisms, including humans. We currently identified >100 single gene knockouts that significantly increase the bacterial chronological lifespan. 

3. *__Synthetic biology and open science__*. While most of the synthetic biology efforts of my team were focused until now on using genetic engineering approaches to address foundational questions as causes and consequences of asymmetric cell division and robustness of cooperation, a shift has occurred in our interests towards more applied research. We host and mentor the award-winning Paris Bettencourt iGEM team and follow-up on their projects. We have developed an open screening approach for antibiotics based on engineering Target-Essential Surrogate E. coli (TESEC) strains, focusing on M. Tuberculosis (MTB) targets. In our preliminary results, we obtained 5 E. coli strains that depend on MTB metabolic genes (e.g, sulphur assimilation, alanine racemase, tryptophan and DAP synthesis) for growth and survival, and initial screening using drug libraries and our automated liquid handling robotic system is ongoing. Other efforts include developing an open diagnostics test for arboviruses (e.g., Zika), identifying structural constraints of RNA folding in vivo and examining the interaction of bacterial cells with blue light.

